### PR TITLE
[10.0][ADD] purchase_line_product_image:

### PR DESCRIPTION
**Pull request by kurniawanlucky**
* Show Product Image at purchase_order_line

### CODE REVIEW

File: purchase_line_product_image/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (LoisRForgeFlow) [inline purchase_line_product_image/__manifest__.py:4]: Please can you add headers to all files following [OCA's template module](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3)

Comment (kurniawanlucky) [reply purchase_line_product_image/__manifest__.py:4]: yes, it's done

**Comment by LoisRForgeFlow** [comment]
Thanks, can you please [check travis](https://travis-ci.org/OCA/purchase-workflow/jobs/294858479)? there are some lints failing.

**Comment by kurniawanlucky** [comment]
@Eficent Thanks for your help

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot. LGTM :+1: 

Thanks for the contribution!

File: purchase_line_product_image/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+Purchase Order Line Product Image for purchases
+==========================================================
```
Comment (lasley) [inline purchase_line_product_image/README.rst]: Please align the lenght of the `=` with the length of the header words (like below)

File: purchase_line_product_image/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
```
Comment (lasley) [inline purchase_line_product_image/README.rst]: This is the old verbiage. Please update to align with [the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker)

File: purchase_line_product_image/__init__.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2017
```
Comment (lasley) [inline purchase_line_product_image/__init__.py]: Your copyright should be one line. It's also worth noting that in a lot of countries, the symbol isn't a valid claim of copyright. I recommend using only the word - `Copyright 2017 My Company`

File: purchase_line_product_image/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2017
+#   @Lucky Kurniawan <kurniawanluckyy@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import purchase_order
```
Comment (lasley) [inline purchase_line_product_image/models/__init__.py]: [Add a carriage return at the end of this line](https://robots.thoughtbot.com/no-newline-at-end-of-file)

File: purchase_line_product_image/models/purchase_order.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    product_image = fields.Binary(
+        "image", related='product_id.product_tmpl_id.image_medium',
+        help="Small-sized image of the product. It is automatically "
```
Comment (lasley) [inline purchase_line_product_image/models/purchase_order.py]: All the attributes but `related` are superfluous and can be removed. Using `related` will bring over all the attributes from the original declaration, so you only need to declare attributes that are being changed 😉

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @kurniawanlucky - comment inline

File: purchase_line_product_image/models/purchase_order.py (hunk 6 above)
Comment (kurniawanlucky) [reply purchase_line_product_image/models/purchase_order.py]: hello @lasley, thanks for your review
can you help me give some example about "so you only need to declare attributes that are being changed"?
thanks

Comment (kurniawanlucky) [reply purchase_line_product_image/models/purchase_order.py]: @lasley remove related and use "@api.onchange"?

Comment (kurniawanlucky) [reply purchase_line_product_image/models/purchase_order.py]: yes, it's done

File: purchase_line_product_image/README.rst [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (lasley) [inline purchase_line_product_image/README.rst]: s/{project_repo}/purchase-workflow

**Review by lasley — APPROVED** [review]
Thanks @kurniawanlucky - minor comment, but doesn't affect usability

File: purchase_line_product_image/README.rst (hunk 7 above)
Comment (kurniawanlucky) [reply purchase_line_product_image/README.rst]: yes, it's done
thanks @lasley

File: purchase_line_product_image/README.rst [hunk 8]
```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
```
Comment (max3903) [inline purchase_line_product_image/README.rst]: @kurniawanlucky License says LGPL here but the files headers and manifest says AGPL.

**Review by max3903 — CHANGES_REQUESTED** [review]

File: purchase_line_product_image/README.rst (hunk 8 above)
Comment (kurniawanlucky) [reply purchase_line_product_image/README.rst]: @max3903 yes, it's done
thanks for your review

**Comment by lasley** [comment]
@max3903 - have your comments been attended to?

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**